### DISCUSSION TITLE
Configuration has section for new item but not for header name?

### DISCUSSION THREAD

**Discussion by Arkhe002** [BenWoodworth/FastCraft#197] [Q&A] [answered]
Server Details
- Server: [1.16.5] 
- FastCraft: v3.2.10

Errors

![image](https://user-images.githubusercontent.com/79669404/132944313-d63e3600-cd8e-40fe-aa77-94d67a09feeb.png)

![image](https://user-images.githubusercontent.com/79669404/132944256-959c13bd-4c6a-4f8f-a17c-96ca353caaf0.png)

![image](https://user-images.githubusercontent.com/79669404/132944214-0252d1e4-78d6-49d2-96ed-1f0db2f72faf.png)

Am I doing wrong?

**Comment by BenWoodworth** [comment, marked as answer]
Hey! You can add data to the item. Something like this:
```yaml
item: minecraft:sunflower{display:{Name:"\"HEADER\""}}
```

I'm not at a computer to test it, you might need single quotes around everything after `item: `, but that's the jist of it :)

**Comment by Arkhe002** [comment]
Thank you it worked. how can i add lore?

**Comment by BenWoodworth** [comment]
Awesome! I'd love to see the custom gui when you're done if you want to share
